docs/tutorials/get-started/blink-an-led.md

```diff
@@ -0,0 +1,442 @@
+You have learned how the GPIO on a Raspberry Pi works, and how to build circuits for LED bulbs.
+
+To make your LED blink periodically, you need to use an SDK.
+
+## Control your robot using the Viam SDKs
+
+Now you're ready to control the LED with code so you don't have to click buttons to turn it on and off.
+
+In this section, you will learn the basics of programming hardware by using either the [Viam Python SDK](https://python.viam.dev/) (software development kit) or the [Viam Go SDK](https://pkg.go.dev/go.viam.com/rdk/robot/client#section-readme) to make your LED blink.
+
+### Install a Viam SDK
+
+Go ahead and install either the [Viam Python SDK](https://python.viam.dev/) or the [Viam Go SDK](https://pkg.go.dev/go.viam.com/rdk/robot/client#section-readme) on your local computer.
+Use which ever programming language you are most comfortable with.
+
+Refer to the appropriate SDK documentation for SDK installation instructions:
+
+- [Viam Python SDK](https://python.viam.dev/)
+- [Viam Go SDK](https://pkg.go.dev/go.viam.com/rdk/robot/client#section-readme)
+
+### Connect your robot to the Viam SDK
+
+The easiest way to get started writing an app/lication with Viam is to navigate to your robot's page on the [Viam app](https://app.viam.com/robots) and select the **Code sample** tab.
+Select your programming language (**Python** or **Golang**), and copy the boilerplate code.
+
+{{% snippet "show-secret.md" %}}
+
+These code snippets import all the necessary libraries and set up a connection with the Viam app in the cloud.
+
+On your local computer, create a file called <file>blink.py</file> or <file>blink.go</file>.
+Paste the boilerplate code from the **Code sample** tab of the Viam app into the file in your code editor, and save the file.
+
+You can now run the code.
+Doing so will ensure that the Viam SDK is properly installed and that the `viam-server` instance on your robot is live.
+
+Run your code:
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+Run the code by typing the following command into the terminal, with the correct path to your file:
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+python3 <INSERT PATH TO YOUR FILE>/blink.py
+```
+
+If you successfully configured your robot and it is able to connect to the Viam app, you should see something like this printed to the terminal after running your program.
+What you see here is a list of the various {{< glossary_tooltip term_id="resource" text="resources" >}} that have been configured on your robot in the Viam app (the board, as well as various built-in services).
+
+<img src="../../img/blink-an-led/python-output.png" alt="A screenshot from the Visual Studio Code command line that prints the output of print(robot.resource_names) when your Raspberry Pi has correctly connected and initialized with the Viam app. The output is an array of resources that have been pulled from the Viam app. Some of these are the Vision Service, Data Manager, and Board." width="500">
```

Comment:
I think better than a screenshot here would be the actual output in code format (same for python )